wordpress-mobile/WordPress-iOS
# Dashboard Cards: Pages card track events

Closes #20445

## Description
Adds tracking for the Pages dashboard card

## Testing Instructions

1. Enable the pages card remote feature flag via the debug menu
2. Run and build the app
3. Verify that `my_site_dashboard_card_shown <blog_id: id, site_type: blog, type: pages>` is tracked ✅
4. Tap on a page item
5. Verify that `my_site_dashboard_card_item_tapped <blog_id: id, site_type: blog, item_type: page, type: pages>` is tracked ✅
6. Go back to the dashboard
7. Tap on the Pages card header
8. Verify that `site_menu_opened <blog_id: id, menu_item: pages, site_type: blog, tap_source: pages_card_header>` is tracked ✅
9. Go back to the dashboard
10. Tap on the ellipsis button
11. Verify that `my_site_dashboard_contextual_menu_accessed <card: pages>` is tracked ✅
12. Tap on the "All Pages" button
13. Verify that `site_menu_opened <blog_id: id, menu_item: pages, site_type: blog, tap_source: pages_card_context_menu>` is tracked ✅
14. Go back to the dashboard
15. Tap on the "Create another page" button
16. Verify that `my_site_dashboard_card_item_tapped <blog_id: id, site_type: blog, item_type: create, type: pages>` is tracked ✅
17. Verify that `editor_page_created <blog_id: id, site_type: blog, tap_source: pages_card>` is tracked ✅
18. Go back to the dashboard
19. Tap on the ellipsis button
20. Tap on the "Hide this" button
21. Verify that `my_site_dashboard_card_hide_tapped <card: pages>` is tracked ✅


## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
